docker/Dockerfile.devito

```diff
@@ -4,14 +4,38 @@
 
 # Base image with compilers
 ARG base=devitocodes/bases:cpu-gcc
+ARG petsc=""
+
+##################  Install PETSc ############################################
+FROM $base as petsc_builder 
+
+ARG petsc
+
+RUN if [ "$petsc" = "true" ]; then \
+        apt-get update && apt-get install -y libopenblas-serial-dev git pkgconf mpich && \
```

Comment:
- git already there
- move libopenblas. pkconf to Dockerfile.cpu/....
- Can't use mpich this will break a lot of config, what's wrong with openmpi?